images/php/7.3/Dockerfile

```diff
@@ -46,14 +46,17 @@ RUN docker-php-ext-install \
   xsl \
   zip
 
+ARG TARGETARCH
+
 RUN cd /tmp \
-  && curl -O https://downloads.ioncube.com/loader_downloads/ioncube_loaders_lin_x86-64.tar.gz \
-  && tar zxvf ioncube_loaders_lin_x86-64.tar.gz \
+  && ARCH=$(if [ "$TARGETARCH" = "arm64" ]; then echo aarch64; else echo x86-64; fi;) \
+  && curl -O https://downloads.ioncube.com/loader_downloads/ioncube_loaders_lin_$ARCH.tar.gz \
```

Comment:
The first line helps us make this single image cross-platform. The `TARGETARCH` build argument is auto-set as `arm64` or `amd64` based on the build, but we need the architecture flags `aarch64` and `x86-64` for the second line, hence my logic.